packages/app-mobile/components/NoteBodyViewer/NoteBodyViewer.tsx

```diff
@@ -67,42 +63,16 @@ export default function NoteBodyViewer(props: Props) {
 		reg.logger().error('WebView error');
 	}
 
-	// On iOS scalesPageToFit work like this:
-	//
-	// Find the widest image, resize it *and everything else* by x% so that
-	// the image fits within the viewport. The problem is that it means if there's
-	// a large image, everything is going to be scaled to a very small size, making
-	// the text unreadable.
-	//
-	// On Android:
-	//
-	// Find the widest elements and scale them (and them only) to fit within the viewport
-	// It means it's going to scale large images, but the text will remain at the normal
-	// size.
-	//
-	// That means we can use scalesPageToFix on Android but not on iOS.
-	// The weird thing is that on iOS, scalesPageToFix=false along with a CSS
-	// rule "img { max-width: 100% }", works like scalesPageToFix=true on Android.
-	// So we use scalesPageToFix=false on iOS along with that CSS rule.
-	//
-	// 2020-10-15: As we've now fully switched to WebKit for iOS (useWebKit=true) and
-	// since the WebView package went through many versions it's possible that
-	// the above no longer applies.
```

Comment:
Any reason why that was removed?

Reply:
We weren't using `scalesPageToFit` anymore, so the `scalesPageToFit` shouldn't apply.

Reply:
https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#scalespagetofit

Nevermind! The default value is `true`! I won't remove the comment!